### ISSUE TITLE
Keine Anmeldung möglich

### ISSUE THREAD

**Issue by JPfeiffer66** [Apollon77/ioBroker.tuya#576]
Hallo!

Kann es sein, dass der Workaround mit der Anmeldung einer älteren SmartLife-Version beim Tuya-Konto nun nicht mehr möglich ist.

Bekomme immer die Meldung "Kontonummer oder Passwort falsch, obwohl die richtig sind. - Die Android-Emu hat auch Zugang zum Internet.

Ist das Problem bekannt?

**Comment by Apollon77** [comment]
Was genau willst du mit der alten App erreichen? Der Adapter macht den Login doch direkt?

**Comment by Apollon77** [comment]
Images bitte nur direkt hochladen, geht per emails nicht

**Comment by JPfeiffer66** [comment]
![Screenshot 2024-05-05 181027](https://github.com/Apollon77/ioBroker.tuya/assets/54292041/b6682d58-7d6f-4d6e-b4a3-23090934689b)

**Comment by Ilovegym66** [comment]
Hallo, scroll doch mal bitte n bisschen tiefer, hast du dich bei Tuya-IOT angemeldet und die Daten da eingetragen?
![image](https://github.com/Apollon77/ioBroker.tuya/assets/36229898/98a5e116-2354-4080-a6b6-db3893e22a50)

**Comment by JPfeiffer66** [comment]
![Screenshot 2024-05-05 181740](https://github.com/Apollon77/ioBroker.tuya/assets/54292041/ebb6acba-25f9-49f1-b9f2-522c1e386aa2)

Ja, habe ich auch. - Auch schon mehrmals geprüft ob die korrekt sind.

**Comment by Ilovegym66** [comment]
Ist der Account auch richtig angelegt und genehmigt worden, schau auf die Bestaetigung, sollte ungefaehr so aussehen??:
![image](https://github.com/Apollon77/ioBroker.tuya/assets/36229898/f751f502-d826-4ad3-9db9-f63b04eaf2fb)

**Comment by JPfeiffer66** [comment]
Ja auch das. - Hatte zwar ursprünglich nur 1 Monat. - Den habe ich jetzt auf 6 Monate erweitert. - Aber leider keine Verbesserung.
![Screenshot 2024-05-05 200700](https://github.com/Apollon77/ioBroker.tuya/assets/54292041/6904c636-fe6f-4d39-9e7b-7928cdce9d67)

**Comment by Ilovegym66** [comment]
Ok, dann den Adapter auf debug stellen, und nochmal den Sync starten, dann das log hier bitte posten.

Bei mir dauert es auch mal bis zu drei-vier Minuten bis der Balken durch ist...

**Comment by JPfeiffer66** [comment]
Guten Morgen!
Der Log-Eintrag bringt leider wenig erhellendes.
![Screenshot 2024-05-06 072947](https://github.com/Apollon77/ioBroker.tuya/assets/54292041/0527826b-45b4-48d8-a48b-edb3a91c42fe)
Habe die Anmeldedaten schon neu eingegeben, Passwort gewechselt... - alles keinen Effekt. :-(

**Comment by Ilovegym66** [comment]
doch, der bringt alles, was man braucht, steht doch da, incorrect Id or password!
Hast du auch bei dem Dev Account EUROPA angewaehlt, und nicht auf China stehen lassen?
Also genau da suchen, da stimmen die credentials nicht.

ps: Log immer komplett von Adapterstart aus posten, hier gehts mal ausnahmsweise aber normal schickt man das komplett und schon garnicht als screenshot.. ist aufm Smartphone unlesbar

**Comment by JPfeiffer66** [comment]
Im Tuya Dev. Account ist GERMANY eingetragen. - Europa gibt es da nicht.

Mir ist aufgefallen, das im Account Center unter Login authorization - service list nichts drin steht. - Geht vlt da was ab?

**Comment by Apollon77** [comment]
Also: Erstmal: Der "Login zum Devices holen" hat mit dem ganzen "iot Cloud" kram nichts zu tun. Die iot Cloud braucht man nur wenn man realtime Date haben will zb auch von Batterie Geräten.

Also lasst uns mal darauf konzentrieren warum der Login nicht geht.

Von daher: Was genau steht denn in der App wenn du dich anmeldest? Steht da vllt (warum auch immer) eine andere regio als Europa? Ggf versuch mal andere Regionen im iorBoker mit deinem account. Die Zugangsdaten bei "Cloud-Usernamer und Passwort" sind die aus deiner Tuya App und NICHT von dieser iot Platform!

**Comment by JPfeiffer66** [comment]
Log-File kommt per eMail.
In der App steht als Kontostandort Deutschland. - Im ioBroker habe ich neben Europa nun auch alle anderen Regionen ohne Erfolg getestst.
Meine Anmeldeinformationen im ioBroker sind identisch mit der in der App.

**Comment by JPfeiffer66** [comment]
Gibt's dazu noch keine neuen Erkenntnisse?

**Comment by Apollon77** [comment]
Mit deinem Log sehe ich auch nichts auffälliges. knn es vllt an Sonderzeichen oder einem zu langen Passwort liegen?

Hast Du Sonderzeichen drin? Wenn ja welche? Wie lang?

**Comment by Apollon77** [comment]
Dann habe ich aktuell keine echte weitere Idee, auch weil Login von mir noch geht mit meinem Accounts und du bisher der einzige bist. Einzige idee ist das du mir die daten per email sendest und ich es selbst versuchen kann nachzustellen

**Comment by JPfeiffer66** [comment]
Habe jetzt alles erdenkliche versucht und ausprobiert. -Auch mit einem zweiten Konto. - Über meinen ioBroker, der auf einem RasPi 4 läuft kann ich mich nicht anmelden. - Natürlich auch schon die Instanz gelöscht und neu installiert.

In der App und auch in Verbindung mit Alexa funktioniert alles einwandfrei.

Mal sehen, ob das Problem irgendwann noch jemand hat oder es sich vlt. von alleine löst. ;-)

**Comment by Apollon77** [comment]
besteht das issue noch?

**Comment by Gemelon** [comment]
Ich habe genau das gleich Problem. Ich habe auch den IoBroker schon zwei mal neu installiert. Das Problem bleibt bestehen. Im Homeassistant funktioniert es. Die Anmeldedaten sind also richtig und grundsätzlich muss es funktionieren.
